### DISCUSSION TITLE
typescript client does not work with my seed file after regenerating.

### DISCUSSION THREAD

**Discussion by Jarrodsz** [geldata/gel#4986] [Q&A]
I seem to have a problem with my generated edgeql-js client.
It does not seem to work for my seed file.

1. I used the remix edgedb example application. 
2. Then modified the schema to add aditional model/types.
3. I ran the create migration and applied migration sucesfully.

When running my seed file ( still the orginal as the remix example application and not modified the existing user and note types )

I get errors. 
Also my IDE shows errors when editing the seed.ts
```
Argument of type '{ title: string; body: string; user: $expr_Select<{ __element__: ObjectType<`${any}`, any, Omit<normaliseShape<{ filter: $expr_Operator<$bool, Cardinality.One>; }, SelectModifierNames>, SelectModifierNames>, ExclusiveTuple>; __cardinality__: Cardinality.Many; }>; }' is not assignable to parameter of type 'never'.
Translation: I was expecting never, but you passed { title: string; body: string; user: $expr_Select<{ __element__: ObjectType<${any}, any, Omit<normaliseShape<{ filter: $expr_Operator<$bool, Cardinality.One>; }, SelectModifierNames>, SelectModifierNames>, ExclusiveTuple>; __cardinality__: Cardinality.Many; }>; }
```

When I copy back the edge client from the remix folder it works again, but ofcourse not for the newly added tables/types.

How can I resolve this?

**Comment by Jarrodsz** [comment]
So the problem was the edgedb remix example is broken. Solved it looking at other repros.

**Reply by raddevon** [reply]
Thank you for letting us know! I'll flag that example for an update to get it working again.
